plugins/es-repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3Service.java

```diff
@@ -94,6 +94,9 @@ private AmazonS3 buildClient(final S3ClientSettings clientSettings) {
         // We do this because directly constructing the client is deprecated (was already deprecated in 1.1.223 too)
         // so this change removes that usage of a deprecated API.
         builder.withEndpointConfiguration(new AwsClientBuilder.EndpointConfiguration(endpoint, null));
+        if(clientSettings.pathStyleAccess) {
+            builder.enablePathStyleAccess();
```

Comment:
```suggestion
        if (clientSettings.pathStyleAccess) {
            builder.enablePathStyleAccess();
```
checkstyle fails because of the missing space